src/styles/main.scss

```diff
@@ -0,0 +1,68 @@
+$width-day: 100px;
+$height-day: 100px;
+$gap-width: 1px;
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  height: 100vh;
+  flex-direction: column;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  box-sizing: border-box;
```

Comment:
You have already set this property for the entire page, I don't think it is necessary to duplicate it.

```suggestion
```

Reply:
Deleting this line fails all tests
![image](https://github.com/mate-academy/layout_calendar/assets/21338158/a6bf9928-8231-49e3-8918-571718c09afa)